tests/std/tests/P0896R4_ranges_range_machinery/test.cpp

```diff
@@ -1886,6 +1886,48 @@ namespace unwrapped_begin_end {
     }
 } // namespace unwrapped_begin_end
 
+namespace closure {
+    // Verify that range adaptor closure capture with the proper value category

```

Comment:
Should this say "closures" plural?

Reply:
Oops - I didn't get all the way from "`_Range_closure`  captures" to "range adaptor closures capture" after changing this to avoid testing the internal mechanism directly.